server/src/yml-objects/AbstractYmlObject.ts

```diff
@@ -32,6 +32,10 @@ export abstract class AbstractYmlObject implements CompletionItem {
     public data?: any;
     /* End of overriden properties. */
 
+    private sourceElementName: string;
```

Comment:
Not sure what this is.

Reply:
You're right. This misses documentation.